package-system/azslc/build_azslc_linux.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+
+cd temp/src
+
+# Call the build and test script 
+python3 test.and.py
+
+exit $?
```

Comment:
The exit call is redundant.
The `python3` invocation return code is the return code of the script